alert.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert Feature</title>
+    <link rel="stylesheet" href="alert.css">
+</head>
+<body>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    
+    <div class="alert-container">
+        <p class="alert-element"> Alerts with white background color and icons</p>
```

Comment:
whay are you providing alert-element class to `p` tag?

Reply:
Earlier I was using flex, now removed it hence, not required.